src/misc/folderDetails.ts

```diff
@@ -1,11 +1,13 @@
 import Command from '@oclif/command';
 import { IAppInfo } from '@rocket.chat/apps-engine/definition/metadata';
 import chalk from 'chalk';
+import * as compareVersions from 'compare-versions';
```

Comment:
You can do this with the `semver` package, no need to bring in yet another library. Right?

Reply:
Yes, you are totally right